Dockerfile

```diff
@@ -76,20 +110,14 @@ LABEL org.opencontainers.image.revision ${GIT_BUILD_HASH}
 
 WORKDIR /
 
-COPY --from=poetry-locker /work/requirements.txt /
-COPY --from=poetry-locker /work/requirements-dev.txt /
```

Comment:
We still need an option to install the dev-dependencies in the final image as that's needed for `ak test-all` (https://github.com/goauthentik/authentik/blob/main/lifecycle/ak#L88) which is used here https://github.com/goauthentik/authentik/blob/main/.github/workflows/release-tag.yml#L24

Reply:
Fixed in d380b396405781f104830f8aa8ceecb6efdc794b